extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -108,6 +128,73 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
     }
     this.subscriptionService_.getDialog().close();
   }
+
+  /**
+   * @return {!Element}
+   * @private
+   * TODO(#37285): add "next story" button to the banner
+   */
+  renderSubscriptionsDialogTemplate_() {
+    return (
+      <div subscriptions-dialog subscriptions-display="NOT granted">
+        <div class="i-amphtml-story-subscriptions-dialog-banner"></div>
+        <div class="i-amphtml-story-subscriptions-dialog-content">
+          <div class="i-amphtml-story-subscriptions-price">
+            {this.element.getAttribute('price')}
+          </div>
+          <div class="i-amphtml-story-subscriptions-title">
+            {this.element.getAttribute('title')}
+          </div>
+          <div class="i-amphtml-story-subscriptions-subtitle-first">
+            {this.element.getAttribute('subtitle-first')}
+          </div>
+          <div class="i-amphtml-story-subscriptions-subtitle-second">
+            {this.element.getAttribute('subtitle-second')}
+          </div>
```

Comment:
What's the benefit of this API instead of supporting `\n` and having one `subtitle` attribute?

Reply:
I think it's major because UX want to control the UI as separate lines so that it won't be a paragraph, and it would be just big and small one line text instead.